src/SourceBuild/content/build.sh

```diff
@@ -175,6 +176,9 @@ while [[ $# > 0 ]]; do
     -use-mono-runtime)
       properties="$properties /p:SourceBuildUseMonoRuntime=true"
       ;;
+    -dev)
+      properties="$properties /p:UseOfficialBuildVersioning=false"
```

Comment:
I wonder why we need a new property for this. Can't we use `OfficialBuildId` or `OfficialBuild` for this instead? Both are already defined Arcade SDK properties.

Reply:
`OfficialBuildId` and `OfficialBuild` aren't the default experience, and we want to keep the VMR as official-build-branded by default at this time.

In the future (as we approach switching to the VMR, have validated testing experiences, etc.) we'll switch the defaults, at which point we can use the traditional Arcade SDK properties.